src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -669,9 +674,23 @@ public class DeleteOldRevisionsResult
             }
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
-
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
+
+            if (conflicted &&
```

Comment:
wrap it up in `ShouldAddConflictRevisionNotification(...)`

Reply:
Done